templated_output/Arkansas_Nuclear_2.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" standalone="no" ?>
```

Comment:
The header file is just so that the xml parser can parse the xml file with xinclude. I've seen some that could not parse it without the header.

This does not get added when the final xml file is parsed either.